src/us/kbase/workspace/test/workspace/ObjectResolverTest.java

```diff
@@ -1701,8 +1692,8 @@ public void searchAsAdminNullNoStartingWorkspace() throws Exception {
 		final WorkspaceUser user = new WorkspaceUser("userfoo");
 		final WorkspaceIdentifier wsi2 = new WorkspaceIdentifier("wsfoo2");
 		final ResolvedWorkspaceID rwsi1 = new ResolvedWorkspaceID(3, "wsfoo", false, false);
-		final ObjectIdentifier pathend = new ObjectIdentifier(wsi2, "objfoo2");
-		final ObjectIDWithRefPath objpath = new ObjectIDWithRefPath(pathend);
+		final ObjectIdentifier objpath = ObjectIdentifier.getBuilder(wsi2).withName("objfoo2")
+				.withLookupRequired(true).build();
```

Comment:
For the sake of your sanity, I hope you were able to do at least some of this via search and replace!

Reply:
Not nearly enough :/